ReactTask/src/tests/SignUpForm.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+import userEvent from '@testing-library/user-event';
+import * as useActionsWithRedux from '../hooks/useActionsWithRedux';
+import { FormSignUp } from '../components';
+import theme from '../theme';
+
+jest.mock('./../redux/patient/patientOperations', () => ({}));
+
+describe('SignUpForm', () => {
+  test('Submit form "Sign up"', async () => {
+    const registerUser = jest.fn((values) => {
+      console.log(values);
```

Comment:
remove  console.log

Reply:
I forgot to delete
Deleted
Better not to fill up at 5 am :)